source/agora/utils/TxBuilder.d

```diff
@@ -228,9 +231,46 @@ public struct TxBuilder
         return this;
     }
 
+    /***************************************************************************
+
+        Sets the unlocker function to sign the inputs
+
+        If not set then the default unlocker using WellKnownKeys is used.
+
+        Params:
+            unlocker = function to sign the inputs of the transaction
+
+        Returns:
+            A reference to `this` to allow for chaining
+
+    ***************************************************************************/
+
+    public ref typeof(this) unlockSigner (Unlock
+        function (in Transaction tx, in OutputRef out_ref) @safe nothrow unlocker) return scope
+        @safe nothrow @nogc pure
+    {
+        this.unlocker = unlocker;
+        return this;
+    }
+
+    public static Unlock signWithGenesis (in Transaction tx, in OutputRef out_ref)
```

Comment:
Missing doc. But how much is this used ? Ideally one could just use `signWithSpecificKey!WK.Keys.Genesis` ?

Reply:
It is used a lot but I created before the `signWithSpecificKey` was created so I think we can use that as you suggest.

Reply:
In fact these are not needed yet so I'll remove from this PR.